docs/_tutorials/reference.md

```diff
@@ -0,0 +1,141 @@
+
+| Argument  | Description  |
+| :---: | :--- |
+| `payload` | All listeners | The unwrapped contents of the incoming event, which varies based on event. This is a subset of the information included in `body` which is detailed below. `payload` is also accessible via the alias corresponding to the method name that the lisener is passed to (`message`, `event`, `action`, `shortcut`, `view`, `command`, `options`) **An easy way to understand what's in a payload is to log it**, or [use TypeScript](/bolt-js/tutorial/using-typescript). |
+| `say` | `message`, `event`, `action` `command` | Function to send a message to the channel associated with the incoming event. This argument is only available when the listener is triggered for events that contain a channel ID (the most common being `message` events). `say` accepts simple strings (for plain-text messages) and objects (for messages containing blocks). `say` returns a promise that will resolve with a [`chat.postMessage` response](https://api.slack.com/methods/chat.postMessage). If you're using an the `action` method, or an event other than `message`, you should [ensure that the event payload contains a channel ID](https://api.slack.com/events).
+| `ack` | `action`, `shortcut`, `view`, `command`, `options` | Function that **must** be called to acknowledge that an incoming event was received by your app. `ack` returns a promise that resolves when complete. Read more in [Acknowledging events](#acknowledging-events)
+| `client` | All listeners | Web API client that uses the token associated with that event. For single-workspace installations, the token is provided to the constructor. For multi-workspace installations, the token is returned by the `authorize` function.
+| `respond` | `shortcut`, `action`, `command` | Function that responds to an incoming event **if** it contains a `response_url`. `respond` returns a promise that resolves with the results of responding using the `response_url`. For shortcuts, `respond` will **only** work for message shortcuts (not global shortcuts).
+| `context` | All listeners | Event context. This object contains data about the event and the app, such as the `botId`. Middleware can add additional context before the event is passed to listeners.
+| `body` | All listeners | Object that contains the entire body of the request (superset of `payload`). Some accessory data is only available outside of the payload (such as `trigger_id` and `authorizations`).
+
+#### Body and payload references
+The structure of the `payload` and `body` is detailed on the API site:
+- `action`: [`body`](https://api.slack.com/reference/interaction-payloads/block-actions) and [`payload`s](https://api.slack.com/reference/block-kit/block-elements)
+- `event`: [`body`](https://api.slack.com/types/event) and [`payload`s](https://api.slack.com/events)
```

Comment:
Nit: `code` links aren't stylized. We could [`try` and `use`](#) the style from GitHub, which turns the text colour blue.

![image](https://user-images.githubusercontent.com/21328/102550829-5ff15400-4073-11eb-95a9-825a38ef3928.png)

Reply:
Good call - I'll update the styling for these elements.